tests/hpc/gnu_compilers_hpc.pm

```diff
@@ -25,7 +25,7 @@ sub run ($self) {
     assert_script_run qq{module av | grep gnu/$version};
     assert_script_run 'module load gnu';
     assert_script_run qq{env |grep "MODULEPATH=/usr/share/lmod/moduledeps/gnu-$version"};
-    if (zypper_call("in gnu$version-compilers-hpc-devel", exitcode => [107]) == 107) {
+    if (zypper_call("in gnu$version-compilers-hpc-devel", exitcode => [0, 107]) == 107) {
```

Comment:
Please check https://progress.opensuse.org/issues/137432#note-3, I have impression that this should be fixed by the update.  

Isn't right time to remove this soft fail logic  (at least for sle12) ?

Reply:
i answered here https://progress.opensuse.org/issues/137432#note-6

Reply:
i think i understand now. Should we keep it to catch any regression? Furthermore, if we remove the check all the other test will start failing until `29418:gnu-compilers-hpc` released.

Reply:
which all the other tests? 
We should not keep it to catch any regressions. If the return code will be !0 we fail since that what the tests do, so we see a  regression.
If we see a failure in sle15 because we still wait for a product fix there, we label that job with a valid bug.

Reply:
honestly it is one or the other way. i dont see the problem with the exitcodes. i remove it however. We can resolve this thread